Easydict/Swift/Feature/Configuration/Configuration.swift

```diff
@@ -402,8 +402,8 @@ class Configuration: NSObject {
         for key in shortcutKeys {
             Defaults.publisher(key)
                 .removeDuplicates()
-                .sink { [weak self] _ in
-                    self?.updateWindowTitlebar()
+                .sink { _ in
+                    EZWindowManager.shared().updateWindowsTitlebarButtonsToolTip()
```

Comment:
we can replace this likes below

```swift
Defaults.publisher(keys: shortcutKeys, options: [])
            .throttle(for: 0.5, scheduler: DispatchQueue.main, latest: true)
            .sink { [weak self] _ in
                EZWindowManager.shared().updateWindowsTitlebarButtonsToolTip()
            }
            .store(in: &cancellables)
```

Reply:
https://github.com/tisfeng/Easydict/pull/500#discussion_r1560374095 We have discussed this logic before, this should fix the duplicate event issue.

Reply:
ok, this code looks a little better.

The last question, does this time threshold 0.5 have any special meaning, or changing it to 0.4 is also okay?

Reply:
The time between repeated events is very short, and 0.5 is a relatively safe value to filter them. Based on my test 0.1 also works well.

Reply:
ok